packages/core/admin/admin/src/pages/Auth/components/ForgotPassword.tsx

```diff
@@ -62,42 +63,41 @@ const ForgotPassword = () => {
               }
             }}
             validationSchema={yup.object().shape({
-              email: yup.string().email(translatedErrors.email).required(translatedErrors.required),
+              email: yup
+                .string()
+                .email({
+                  id: translatedErrors.email,
+                  defaultMessage: 'This is not a valid email.',
```

Comment:
Should we update this here? https://github.com/strapi/strapi/blob/56142c70f8b51d644d00a5ed70f1795637b29d91/packages/core/email/admin/src/translations/en.json#L22